### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Vinceeeeeent** [Backdash/MonikaModDev#1054]
I did try to start it 4 times by now, this message always shows up. 

[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/valentines.rpy", line 168, in script
  File "game/valentines.rpy", line 171, in <module>
  File "game/event-handler.rpy", line 129, in addEvent
    raise EventException("Syntax error in conditional statement for event '" + event.eventlabel + "'.")
EventException: EventError: Syntax error in conditional statement for event 'monika_valentines_start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/valentines.rpy", line 168, in script
  File "C:\Program Files\Steam\steamapps\common\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Program Files\Steam\steamapps\common\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/valentines.rpy", line 171, in <module>
  File "game/event-handler.rpy", line 129, in addEvent
    raise EventException("Syntax error in conditional statement for event '" + event.eventlabel + "'.")
EventException: EventError: Syntax error in conditional statement for event 'monika_valentines_start'.

Windows-7-6.1.7601-SP1
Ren'Py 6.99.12.4.2187
Monika After Story 0.7.1
[/code]

**Comment by ThePotatoGuy** [comment]
Try full delete and reinstall.

**Comment by ThePotatoGuy** [comment]
Closing in favor of #1053
